### DISCUSSION TITLE
[@types/react-native-auth0] No custom URL parameters in authorize function

### DISCUSSION THREAD

**Discussion by bombillazo** [DefinitelyTyped/DefinitelyTyped#59962] [Issues with a @types package] [answered]
While not explicitly specified in the documentation, one can pass custom URL parameters to the `auth0.WebAuth.authorize` function as seen [here ](https://community.auth0.com/t/custom-authorize-params-extraparams/38412/8) and [here](https://community.auth0.com/t/passing-initialscreen-to-lock-from-webauth/6906/2).

Currently there is no typing option to include custom parameters:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/94d284025fbc1c0f55f75c55f9ec7758260d672f/types/react-native-auth0/index.d.ts#L202-L213

[a comment by an automated account is not shown]

**Comment by leoafarias** [comment]
Is that for the "mode" or a custom parameter in general? Do a PR and I will review.

**Reply by bombillazo** [reply]
For custom parameters in general, you can technically add any key:value pair to the `AuthorizeParams`  object, and Auth0 will append it to the URL parameters of the authorization page.

**Comment by bombillazo** [comment, marked as answer]
PR created: #59965
